src/debug.cpp

```diff
@@ -91,6 +91,28 @@ extern bool test_mode;
 /** Set to true when any error is logged. */
 static bool error_observed = false;
 
+/** If true, debug messages will be captured. */
```

Comment:
Maybe expand this comment to explain that this is used to test debugmsg calls in the unit tests.

Reply:
done